helix-term/src/ui/picker.rs

```diff
@@ -612,30 +611,34 @@ impl<T: 'static> Component for Picker<T> {
                 surface.set_string(inner.x.saturating_sub(2), inner.y + i as u16, ">", selected);
             }
 
-            let formatted = (self.format_fn)(option);
-
+            let spans = (self.format_fn)(option);
             let (_score, highlights) = self
                 .matcher
-                .fuzzy_indices(&formatted, self.prompt.line())
+                .fuzzy_indices(&String::from(spans.clone()), self.prompt.line())
```

Comment:
why the clone?

Reply:
Bc. we iterate the spans below.

Reply:
I believe `String::from` is already copying, so the clone here is adding a second copy. Does it give any error without the clone?

Reply:
There was no implementation `From<&Spans> for String`, only a non-reference conversion that consumes the Spans. 
I added it so we can save a clone.